RedRover:
  mpc_node:
    ros__parameters:
      num_timesteps: 5
      tf: 1.0
      dt: 0.2 # should dt be smaller? getting occassional error with time tolerance of robotdatapy ArrayData
      # dt: 0.05 # should dt be smaller? getting occassional error with time tolerance of robotdatapy ArrayData
      x_bounds: 
        n: 3
        xb0: [-10.0, 10.0]
        xb1: [-10.0, 10.0]
        xb2: [-.inf, .inf]
      u_bounds: 
        n: 2
        ub0: [-1.0, 1.0]
        ub1: [-1.5, 1.5]
